### PR TITLE
OntoPBM-Watershed

### PR DESCRIPTION
**Pull request by ilin-t**
Added permanent identifier for the watershed ontology OntoPBM-Watershed

### CODE REVIEW

File: ontopbm-aquatic/.htaccess [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Header set Access-Control-Allow-Origin *
 Header set Access-Control-Allow-Headers DNT,X-Mx-ReqToken,Keep-Alive,User-Agent,X-Requested-With,If-Modified$
 Options +FollowSymLinks
-RewriteEngine on
+RewriteEngine On
```
Comment (davidlehn) [inline ontopbm-aquatic/.htaccess]: I'm guessing you tested this?  Docs just use lowercase `on`:
https://httpd.apache.org/docs/2.4/mod/mod_rewrite.html#rewriteengine

File: ontopbm-watershed/.htaccess [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Header set Access-Control-Allow-Origin *
+Header set Access-Control-Allow-Headers DNT,X-Mx-ReqToken,Keep-Alive,User-Agent,X-Requested-With,If-Modified$
+Options +FollowSymLinks
+RewriteEngine on
+RewriteRule ^$http://source.ijs.si/itolovski/OntoPBM/raw/master/OntoPBM-Watershed.owl [R=302,L]
```
Comment (davidlehn) [inline ontopbm-watershed/.htaccess]: Looks like this is missing whitespace after the pattern.

File: ontopbm-aquatic/.htaccess (hunk 1 above)
Comment (ilin-t) [reply ontopbm-aquatic/.htaccess]: It was an oversight, should be lowe case

**Comment by ilin-t** [comment]
I created new pull request with these issues fixed.

**Pull request merged**